app/assets/stylesheets/navbar.scss

```diff
@@ -1,172 +1,145 @@
+        @media screen and (max-width: $screen-lg) {
+          display: none;
         }
       }
 
-      .dropdown-content {
+      .nav-menu-expand {
         display: none;
-        position: absolute;
-        left: auto;
-        margin-top: 1.2rem;
-        background-color: $color-white;
-        box-shadow: 0 0.1rem 0.25rem $color-brand-light;
-        z-index: 1;
-
-        .nav-link {
-          display: block;
-          margin: 0 1rem;
-          color: $ink;
-          font-size: 1rem;
-
-          i {
-            font-size: 1.5rem;
-          }
 
-          &:hover {
-            color: $color-secondary;
-          }
+        @media screen and (max-width: $screen-lg) {
+          display: inline-block;
         }
       }
+    }
+  }
 
-      &:hover {
-        .dropdown-button {
-          color: $color-brand
-        }
+  .nav-links {
+    .link {
+      color: $color-primary-0;
+      font-weight: 600;
+      letter-spacing: -1px;
+      font-size: 1.2rem;
+      transition: border-bottom-color .25s;
+      border-bottom: 3px solid;
 
-        .dropdown-content {
-          display: block;
-        }
+      &:hover {
+        color: $color-primary-1;
       }
     }
+  }
 
-    .nav-mobile-expand {
-      display: none;
-      font-size: 1.5rem;
-    }
+  .nav-authentication-btns {
+    gap: 0.5rem;
+  }
 
+  .nav-menu-expand, .nav-menu-collapse {
+    font-size: 1.5rem;
+    line-height: 0;
+    padding: 0.6rem 1.5rem;
+    width: auto;
   }
-}
 
-@media screen and (max-width: $screen-md) {
-  .navbar {
-    .nav-left .nav-links {
-      display: none;
+  .nav-menu {
+    position: fixed;
+    right: 0;
+    top: 0;
+    z-index: 100;
+    width: 300px;
+    height: 100vh;
+    padding: 0.75rem 1rem;
+    background-color: white;
+    box-shadow: 0 0 2px 0 $color-primary-0;
+    align-items: flex-end;
+    gap: 2rem;
+    display: none;
+
+    @media screen and (min-width: $screen-lg) {
+      display: none !important;
     }
 
-    .nav-center {
-      .nav-brand {
-        margin-top: 1rem;
-        line-height: 0.8;
-        font-size: 6rem;
-      }
-    }
+    .nav-links {
+      gap: 1rem;
+      width: 100%;
 
-    .nav-right {
-      .nav-dropdown {
-        display: none;
-      }
+      .link {
+        width: fit-content;
+        border-bottom-color: $color-complementary-0;
 
-      .nav-mobile-expand {
-        font-size: 3rem;
-        display: block;
+        &:hover {
+          border-bottom-color: $color-complementary-1;
+        }
       }
     }
 
-    .nav-link {
-      padding: 1rem 0;
+    .nav-authentication-btns {
+      width: 100%;
+      justify-content: center;
     }
 
-    &.expanded {
-      align-items: flex-start;
-
-      .nav-left, .nav-links {
-        flex-flow: column;
-        align-items: flex-start;
-      }
-
-      .nav-left .nav-links {
-        display: flex;
-      }
+    &-background {
+      position: fixed;
+      top: 0;
+      left: 0;
+      z-index: 50;
```

Comment:
not saying no to using 50 and 100. Not actually sure what best practice for z-index values